src/RapidezServiceProvider.php

```diff
@@ -275,6 +275,25 @@ protected function bootMacros(): self
                 ->squish();
         });
 
+        Vite::macro('getPathsByFilenames', function ($filenames) {
+            /** @var \Illuminate\Foundation\Vite $this */
+            $filenames = is_array($filenames) ? $filenames : func_get_args();
+            $manifest = $this->manifest($this->buildDirectory);
+
+            return array_filter(
+                array_map(
+                    function ($filename) use ($manifest) {
+                        foreach ($manifest as $path => $asset) {
+                            if (Str::endsWith($asset['name'] ?? '', $filename) || Str::endsWith($path, $filename)) {
```

Comment:
Instantsearch works with `.vue.js` files, checking the name ensures an import with `.vue` in the name still gets caught